docs/examples/node_postprocessor/rankGPT.ipynb

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
<ul><li>This demo *integrates</li><li>Is your [RankGPT] supposed to be a link?</li><li>Either way would link to the paper</li><li>can you add one sentence describing what RankGPT is? like the core idea of using LLMs for reranking? </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10054/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>